### DISCUSSION TITLE
low nFeature_RNA in specific cluster

### DISCUSSION THREAD

**Discussion by dahun73** [satijalab/seurat#6017] [Q&A] [answered]
Hello :)

I'm trying to analyze scRNA-seq data for mouse brain. During processing, however, I have one question about this situation.

![image](https://user-images.githubusercontent.com/52641420/170927530-df5c7dbb-0d4a-4d4c-a2f2-1cd50ba0aaf8.png)

I checked nFeature_RNA for each seurat cluster. Some of clusters have few nFeature_RNA(clu2, clu4, clu10, clu13). It means cells in those clusters have few detected genes. Of course, these cluster could be rare cell type and they could be detected for small number of genes. However, it is normal condition of mouse brain for E15. I think it is not normal situation.....

Also, I checked cluster markers for those clusters. 
![image](https://user-images.githubusercontent.com/52641420/170928024-f9609f77-f292-4615-ad6e-179055bffd64.png)
This picture is clu2 marker genes for one of those clusters. 'avg_log2FC' is lower than other cluster and marker genes are not informative(I mean that it is hard to decide which cell type it is). 

In this case, is it problem of sample quality? Or is it normal...? Is there any ideas handling these low detected genes?(such as QC or changing filtering parameter, ...).

+) I filtered nFeature_RNA from 200 to 4000.

```Control1 <- subset(Control1, subset = nFeature_RNA > 200 & nFeature_RNA < 4000 & percent.mt < 10)```

Thank you in advance!!

**Comment by timoast** [comment, marked as answer]
It's likely that these are low quality cells

**Comment by f6v** [comment]
The best answer is: it depends. Can you assign a meaningful biological identity to this cluster? Any pathways that are enriched among the DE genes? What about the cell cycle scores? Have you checked against known tissue dissociation signatures? 

If it doesn't look like anything you'd expect or trust, then it's likely a technical artefact.

**Reply by dahun73** [reply]
> The best answer is: it depends. Can you assign a meaningful biological identity to this cluster? Any pathways that are enriched among the DE genes? What about the cell cycle scores? Have you checked against known tissue dissociation signatures?
> 
> If it doesn't look like anything you'd expect or trust, then it's likely a technical artefact.

Thank you for kind explanation.
You are right. When I analyze other samples, some of clusters have few detected genes. I think it is cell-type specific(maybe it is artifact or low quality cells). So, I should check more and more to find something meaningful.


Thanks!
